internal/e2e/simple/simple_test.go

```diff
@@ -54,6 +64,12 @@ sh_binary(
 )
 -- subdir/subdir.sh --
 printf "Hello subdir!"
+-- test_simple_test_failing.sh --
+echo "I'm a failing test."
+exit 1
+-- test_simple_test_passing.sh --
+echo "I'm a passing test."
```

Comment:
if you invoke `env` in here are there any environment variables that differ between a `test` run and a `coverage` run?

Reply:
I tried with printenv in both tests and all the environment variables were the same. I can't find a better way to test it than what I did.

Reply:
I think this is as good as it is going to get